app/page.tsx

```
@@ -1,9 +1,14 @@
-import Bridge from '@/components/bridge'
+'use client'
+
+import React, { useState } from 'react'
+import MainModal from '@/components/main-modal'
 
 export default function Home() {
+  const [modalOpen, setModalOpen] = useState<boolean>(true)
```

ESLint

[error] 'modalOpen' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)